src/reducers/images.js

```diff
@@ -18,15 +18,19 @@ import {
   UPDATE_STRATEGY,
   RESET,
   UPDATE_TEXT,
-  UPDATE_TEXT_COLOR
+  UPDATE_TEXT_COLOR,
+  UPDATE_TEXT_POSITION
 } from '../constants/action-types';
+import { startTimer } from '../lib/timer';
```

Comment:
This is imported but never used.

Reply:
I'm not sure what this does, it's been in the file, should I delete it?

Reply:
Yep, delete!

Reply:
Done